### PR TITLE
Gateblu

### PR DESCRIPTION
**Pull request by sqrtofsaturn**
Adding Gateblu to homebrew-cask

### CODE REVIEW

File: Casks/gateblu.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'gateblu' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://s3-us-west-2.amazonaws.com/gateblu/Gateblu.dmg'
```
Comment (vitorgalvao) [inline Casks/gateblu.rb]: Needs a `# amaonaws.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by sqrtofsaturn** [comment]
Updated.

**Comment by vitorgalvao** [comment]
You’ve added more commits. Need help?

**Comment by sqrtofsaturn** [comment]
Do you have any suggestions on how the commits should look? We squashed the
commits, which created a new commit of the squashed ones.

**Pull request closed without merging**